src/main/java/org/jetbrains/bsp/bazel/server/logger/BuildClientLogger.java

```diff
@@ -0,0 +1,25 @@
+package java.org.jetbrains.bsp.bazel.server.logger;
+
+import ch.epfl.scala.bsp4j.BuildClient;
+import ch.epfl.scala.bsp4j.LogMessageParams;
+import ch.epfl.scala.bsp4j.MessageType;
+
+public class BuildClientLogger {
+
+  private final BuildClient buildClient;
+
+  public BuildClientLogger(BuildClient buildClient) {
+    this.buildClient = buildClient;
+  }
+
+  public void logError(String errorMessage) {
+    LogMessageParams params = new LogMessageParams(MessageType.ERROR, errorMessage);
+    buildClient.onBuildLogMessage(params);
+    throw new RuntimeException(errorMessage);
```

Comment:
Throwing an error inside a logging method looks suspicious (inb4 I know it used to be so even before the refactoring)

Reply:
added TODO